app/controllers/accounts_controller.rb

```diff
@@ -15,7 +15,7 @@ def search
 
   # Does a lookup from the account service in production mode, otherwise examines local database for users
   def lookup_account
-    return AccountService.call(sunetid: params[:id]) if Rails.env.production?
+    return AccountService.call(sunetid: params[:id]) unless Rails.env.development?
```

Comment:
Going to explicitly mock out the Account Service in tests, so the fake Account Service is only needed in development.